.yarnrc.yml

```diff
@@ -5,8 +5,10 @@ enableGlobalCache: true
 nodeLinker: node-modules
 
 npmAuditExcludePackages:
-  - angular
-  - bootstrap
-  - bootstrap-sass
+- jquery
+# pending | moderate | GHSA-rmxg-73gg-4p98 | jquery >=1.12.3 <3.0.0 | 2.2.4 brought in by manageiq-ui-classic@workspace:.
+# pending | moderate | GHSA-gxr4-xjj5-5px2 | jquery >=1.2.0 <3.5.0  | 2.2.4, 3.4.1 brought in by manageiq-ui-classic@workspace:., patternfly@npm:3.59.5
+# pending | moderate | GHSA-6c3j-c64m-qhgq | jquery >=1.1.4 <3.4.0  | 2.2.4 brought in by manageiq-ui-classic@workspace:.
+# pending | moderate | GHSA-jpcq-cgw6-v4j6 | jquery >=1.0.3 <3.5.0  | 2.2.4, 3.4.1 brought in by manageiq-ui-classic@workspace:., patternfly@npm:3.59.5
```

Comment:
Something weird happened with this revert. the angular and bootstrap entries need to stay.

Reply:
@jrafanie It's my fault - when I created the 4.9.1 PR, I accidentally removed the comments, which prevented the fix...let me get in a PR to undo that.